test/spinach/runner_test.rb

```diff
@@ -119,7 +119,50 @@
 
         runner.run.must_equal true
         feature.filename.must_equal filename
-        feature.line.must_equal line
+        feature.lines.must_equal [line]
+      end
+    end
+
+    describe 'when lines set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.